.github/CODEOWNERS

```diff
@@ -1,4 +1,3 @@
 @bowenlan-amzn
-@Angie-Zhang
 @getsaurabh02
 @praveensameneni
```

Comment:
```suggestion
@SuZhou-Joe
```

Reply:
This is the wrong format, should be a `* ...list`.